sumologic/resource_sumologic_password_policy.go

```diff
@@ -0,0 +1,157 @@
+package sumologic
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func resourceSumologicPasswordPolicy() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceSumologicPasswordPolicyCreate,
+		Read:   resourceSumologicPasswordPolicyRead,
+		Update: resourceSumologicPasswordPolicyUpdate,
+		Delete: resourceSumologicPasswordPolicyDelete,
+
```

Comment:
We should allow importing existing password policy. Don't forget to add import acceptance test as well.